piq/gmsd.py

```diff
@@ -43,9 +43,11 @@ def gmsd(prediction: torch.Tensor, target: torch.Tensor, reduction: str = 'mean'
         https://arxiv.org/pdf/1308.3052.pdf
     """
 
-    _validate_input(input_tensors=(prediction, target), allow_5d=False, scale_weights=None)
+    _validate_input(
+        input_tensors=(prediction, target), allow_5d=False, scale_weights=None, data_range=data_range)
     prediction, target = _adjust_dimensions(input_tensors=(prediction, target))
 
+    # Rescale to [0, 1] range
```

Comment:
This comment is confusing. The operation below does not always scale to the [0, 1] range because a user can provide `data_range` that would scale this particular `prediction` and `target` pair to, let's say, [0, 0.5] range. Please update the description.